Dockerfile-insights-operator-runtime

```diff
@@ -0,0 +1,15 @@
+FROM redhat/ubi9-minimal
+
+# adding tar so that `kubectl cp` can work and fetch the gathered data.
+# For the final integration, the base image would be registry.ci.openshift.org/ocp/4.17:base-rhel9
+# that already contains tar
+RUN microdnf update -y && microdnf -y install \
+    tar
+
+COPY ./bin/insights-operator /usr/bin/
+COPY config/pod.yaml /etc/insights-operator/server.yaml
+COPY manifests /manifests
+# Do not start this instance of the insights-operator 
+# as it would conflict with the one already installed by OLM.
+# We just sleep so that the gather operation can be executed on that instance
+ENTRYPOINT ["sleep", "360000"]
```

Comment:
This file is just temporary right? In sense there is no plan to merge it in the repository. Do I understand correctly? Same for the script below.

Reply:
yes, that's correct.